apps/web/src/components/editor/action-bar.tsx

```diff
@@ -544,7 +556,17 @@ function Tab(props: TabProps) {
           if (e.button == 0) onFocus();
         }}
       >
-        <Icon size={16} color={isActive ? "accent-selected" : "icon"} />
+        <Icon
+          data-test-id={
+            isUnsaved
+              ? "note-icon-unsaved-" + title
+              : "note-icon-saved-" + title
```

Comment:
Use id here as title can be the same for notes causing issues.

Also it should be `tab-icon` not `note-icon`